docs/install-usage-fleetCLI.md

```diff
@@ -0,0 +1,131 @@
+  * This applies for `fleet.yaml`, Helm charts and manifests. For example, `fleet apply my-bundle ./manifests`.  
+* `fleet target`: Reads a bundle file and evaluates which clusters would receive it, based on selectors and targeting rules such as `targets`, `targetOverrides`, `clusterGroups`, and `label` selectors.
+  * For example, `fleet target my-bundle ./manifests.`
+* `fleet deploy`: takes the output of `fleet target`, or a dumped `bundledeployment`/content resource and deploys it to a cluster, just like fleet-agent would. You can use it in these scenarios:
+  * `fleet apply -o - name ./folder` to check the YAML of the bundle before creating it. For more information, refer to [Convert a Helm chart into a bundle.](#convert-a-helm-chart-into-a-bundle)
+  * Use with a target to debug selectors and verify which downstream clusters are targeted.
+  * `fleet deploy --dry-run` to print the resources that would be deployed, but does not apply them.  
+
+![A diagram explaining how fleet CLI key commands work.](../static/img/fleetCLI-key-components.svg)
+
+For more information, refer to [Bundle Lifecycle With the CLI](ref-bundle-stages.md#examining-the-bundle-lifecycle-with-the-cli).
+
+## Deploy a Sample Bundle Using Fleet CLI
+
+You can deploy workloads without using GitRepos by applying them locally with the CLI. For example, consider using the [Fleet examples repository](https://github.com/rancher/fleet-examples).
+
+```bash
+git clone https://github.com/rancher/fleet-examples  
+cd fleet-examples/single-cluster
+```
+
+Apply it to the current cluster:
+
+```bash
+fleet apply -o my-cool-bundle manifests
+```
+
+This creates a Bundle resource in the namespace.
+
+## Convert a Helm Chart into a Bundle
+
+You can use the Fleet CLI to convert a Helm chart into a bundle.
+
+For example, you can download and convert the "external secrets" operator chart like this:
+
+```bash
+cat > targets.yaml <<EOF
+targets:
+- clusterSelector: {}
+EOF
+
+mkdir app
+cat > app/fleet.yaml <<EOF
+defaultNamespace: external-secrets
+helm:
+  repo: https://charts.external-secrets.io
+  chart: external-secrets
+EOF
+
+fleet apply --compress --targets-file=targets.yaml -n fleet-default -o - external-secrets app > eso-bundle.yaml
+
+kubectl apply -f eso-bundle.yaml
+```
+
+Make sure you use a cluster selector in `targets.yaml`, that matches all clusters you want to deploy to.
+
+The blog post on [Fleet: Multi-Cluster Deployment with the Help of External Secrets](https://www.suse.com/c/rancher_blog/fleet-multi-cluster-deployment-with-the-help-of-external-secrets/) has more information.
+
+
+## **Troubleshooting**
+
+If the bundle is not ready:
+
+* Check if `fleet-controller` and `fleet-agent` pods are running.  
+* Make sure the `fleet-local` cluster is registered.
```

Comment:
This could be extended to all target clusters in general, as users may run the Fleet CLI with clusters other than the local one in mind:
```suggestion
* Make sure your target clusters are registered.
```

... by the way, we could explain in a few words how to check this and/or link to more information on cluster registration, for instance https://fleet.rancher.io/ref-registration#fleet-agent-is-registered-watches-for-bundledeployments 
We could eventually add troubleshooting info to that page, but checking that a Fleet agent is running on a given cluster is a good (albeit insuficient) first step.